src/mlagility/analysis/status.py

```diff
@@ -133,6 +142,8 @@ def print_invocation(
 
     print(f"{ident}\tInput Shape:\t{input_shape}")
     print(f"{ident}\tHash:\t\t" + invocation_hash)
+    print(f"{ident}\tBuild Name:\t" + build_name)
```

Comment:
Discussion: Does it make sense to print the build name even though the model has not been built (e.g. when using `--analyze-only`)? This is the current behavior of this MR.

If that doesn't make sense, why not simply add the build name as part of the status?

```
Workloads discovered during profiling:

two_input_shapes.py:
        model
                Model Type:     Pytorch (torch.nn.Module)
                Class:          EfficientNet (<class 'timm.models.efficientnet.EfficientNet'>)
                Location:       /home/dhnoronha/.local/lib/python3.8/site-packages/timm/models/efficientnet_builder.py, line 472
                Parameters:     1,677,128 (3.2 MB)

                Workload 1 (executed 1x)
                Input Shape:    'Positional Arg 1': (1, 3, 28, 28)
                Hash:           4c4868c4
                Status:         Successfully benchmarked on Intel(R) Xeon(R) CPU @ 2.20GHz (ort v1.14.1)
                                Build name:     <BUILD_NAME_HERE>
                                Mean Latency:   6.218   milliseconds (ms)
                                Throughput:     160.8   inferences per second (IPS)


                Workload 2 (executed 1x) - <BUILD_NAME_HERE>
                Input Shape:    'Positional Arg 1': (1, 3, 224, 224)
                Hash:           cbdf10c5
                Status:         Successfully benchmarked on Intel(R) Xeon(R) CPU @ 2.20GHz (ort v1.14.1)
                                Build name:     <BUILD_NAME_HERE>
                                Mean Latency:   27.141  milliseconds (ms)
                                Throughput:     36.8    inferences per second (IPS)
```

Reply:
If I had to choose 1 between `build_name` and `cache_dir` I would pick `cache_dir` as that is more actionable and also includes the build name.

Reply:
Agreed. `build_dir` instead of `build_name` + `build_dir` sounds like a good option.

Reply:
Sounds good. 
So we are adding only one line instead of two.
`Build dir: cache_dir + build_name` which will be the path where all the files for that build can be found.

Reply:
![image](https://github.com/groq/mlagility/assets/12814146/a8dc4121-222a-45c6-a0d4-1d718c3e2476)